<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalSlotsWidget</class>
 <widget class="QWidget" name="SignalSlotsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>信号与槽的5种方式</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btn_Max">
     <property name="text">
      <string>最大化显示(1:SIGNAL/SLOT)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_Normal">
     <property name="text">
      <string>正常显示(2:使用函数地址)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_Min">
     <property name="text">
      <string>最小化显示(3:使用设计师界面-&gt;转到槽)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_close">
     <property name="text">
      <string>关闭窗口(4:设计师界面-&gt;信号与槽函数的编辑器)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_changeTitle">
     <property name="text">
      <string>修改窗口的标题(5:Lmabda表达式)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>SignalSlotsWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
